bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -547,6 +548,27 @@ impl Client {
             Ok(user_profile)
         })
     }
+
+    /// Registers a notification handler on the delegate if required
+    /// the sliding sync requires to have registered m.room.member with value
```

Comment:
Are these supposed to be separate sentences?

```suggestion
    /// Registers a notification handler on the delegate if required.
    ///
    /// The sliding sync requires to have registered m.room.member with value
```